src/components/PreviewConcept/PreviewConceptLightbox.tsx

```diff
@@ -48,6 +48,39 @@ const closeButtonStyle = css`
   margin-top: -15px;
 `;
 
+export const getVisualElement = async (element: VisualElement | string) => {
+  const visualElement = typeof element === 'string' ? parseEmbedTag(element) : element;
```

Comment:
Vil helst ikke gjøre dette, men vil også kunne gjenbruke funksjonen slik at alle steder som skal vise VisueltElement kan få url'ene fra samme sted. Konkrete forslag til forbedring, eller forklaring på hvorfor den ene typen visueltElement er enn streng tas imot med takk.

Reply:
Apiet tar imot en streng men returnerer et objekt med language og visualElement som er strengen som sendes inn. Det er kanskje grunnen til at du må ha dette, dersom funksjonen skal støtte begge formata.

Reply:
Ja, det gir mening. Skulle helst sett at den strengen inneholdt de feltene som var gjort reqiured i VisualElement interfacet slik at jeg kunne parset og satt type på den, ser ut som om det er flere ting som depender på det feltet som er required. `resource_id` er required men kommer ikke med alle typene fra api'et. @ingeborgsteel du satte deg kanskje litt inn i dette da du jobbet med det?

Reply:
Beste hadde vert om vi hadde 2 forskjellige felt visualElementEmbed: string og visualElement: VisualElementType for eksempel slik at vi ikke dyttet to forskjellige typer inn i samme variabel i hytt og pine